lib/model/init.js

```diff
@@ -0,0 +1,46 @@
+const inquirer = require('inquirer');
```

Comment:
The lib/model module is to have data structures that help us complete the local computation if required. The lib/model does not deal with any input / output related work.